### ISSUE TITLE
How to use  @ in (text)critical apparatus

### ISSUE THREAD

**Issue by stefaniasalvadori** [dariok/karlstadt-issues#172]
Wir haben in der Formatierung der LD (=EE103) das Zeichen " @ " (Spatium+@+Spatium - NICHT KURSIV) eingefügt.
Bitte nochmals erklären, wann das Zeichen verwenden werden muss.

Bisher nur im Fall eine witness ohne text in längeren Anm. mit texten und Sigeln
So z.B.:

**ac** Liberum arbitrium _A5;_ lberum ar _a1;_ **@** _korrigiert a2;_  Eckius <contra?> 11 conclusio<nem> doc Corol<ostadii> de libero arbitrio  _a2 AuRd-Gl;_

**ad** sactissimum _a1;_ @ _korrigiert a2_

**af** Invocavit auxilium _A5;_ @ _fehlt B;_ auxilium _a1;_ 

NICHT ABER im Fall einfache Anm. 
**av**  _fehlt a1_

NICHT ABER wenn ein "Zitat" steht, auch wenn das Zitat einem kursivierten Text folgt:
**bc** _über der Zeile hinzugefügt_ opus _a2_ 
?? Wie formatieren wir diese Anmerkungen??? Einfach als Text? oder Standard? + Kursivierungen?

**Comment by dariok** [comment]
Genau, » @ « wird verwendet, wenn in einer textkritischen Anmerkung bei einem Textzeugen kein Originaltext steht: bei **ad** würde ja in voller Länge sonst stehen »sactissimum *a1;* sanctissimum *a2*«

Bei **av** fehlt kein Originaltext im der Anmerkungstext, dies wird auch weiterhin korrekt umgesetzt. Auch bei **bc** muß nichts geändert werden, denn der Originaltext ist ja in recte vorhanden.

Den Fall **af** müßte ich mir nochmal anschauen; »*fehlt B*« müßte eigentlich korrekt umgesetzt werden.

**Comment by stefaniasalvadori** [comment]
In Ordnung - Danke!

Bei **ad** darf ich »sactissimum _a1; korrigiert in_ sanctissimum _a2_« schreiben?
a2 ist eine Korrektur von a1; sie sind nicht zwei Vorlagen, sondern nur eine mit Korrekturen.

Wenn das problematisch ist, dann bleibe ich bei Deinem Vorschlag.
